app/src/main/java/io/github/trojan_gfw/igniter/MainActivity.java

```diff
@@ -226,6 +226,8 @@ protected void onResume() {
             updateViews(serviceInstance.getState());
             clashSwitch.setChecked(serviceInstance.enable_clash);
         }
+//        ContactsContract.CommonDataKinds.Phone
```

Comment:
I think we should drop the comment.

Reply:
Sorry, I didn't check the changes of "MainActivity.java". Actually, I pulled files from upstream and only modified "activity_main.xml".